test/delighted_test.rb

```diff
@@ -170,3 +170,39 @@ def test_listing_all_survey_responses_expand_person
     assert_equal({ :email => 'foo@bar.com' }, survey_responses[1].person.to_hash)
   end
 end
+
+class Delighted::UnsubscribesTest < Delighted::TestCase
+  def test_listing_unsubscribes
+    uri = URI.parse("https://api.delightedapp.com/v1/unsubscribes")
+    headers = { 'Authorization' => "Basic #{["123abc:"].pack('m0')}", "Accept" => "application/json", 'User-Agent' => "Delighted RubyGem #{Delighted::VERSION}" }
+    example_unsub = {person_id: '4945', email: 'foo@example.com', name: nil, unsubscribed_at: 1440621400}
```

Comment:
Since we are supporting Ruby 1.8, will need to change these Hash notations to rocket style.